docs/program/apis/disconnect.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Session Connectivity: Network Disconnection during a Session"
+linkTitle: "Network Disconnection"
+weight: 20
+type: "docs"
+description: "How Viam handles losing connection to the internet during a robot session."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session"]
+---
+
+What happens when your robot loses its connection to the internet and can no longer talk to the Viam cloud through [the Viam app](https://app.viam.com).
+
+### How does `viam-server` work if your robot loses connectivity during a session?
+
+If the robot loses connectivity, `viam-server` will timeout and end any client [*sessions*](/program/apis/sessions/) on this robot.
+When a session times out, any active commands will be cancelled, stopping any moving parts, and no new commands will be able to reach the robot until the connection is restored.
+
+
+To disable the default behavior here and manage resource timeout and reconfiguration over a networking session yourself, follow [these instructions](/program/apis/sessions/).
+[Disable the default behavior](/program/apis/sessions/#disable-default-session-management) of session management, then utilize [Viam's SDKs](/program/) in your code to make calls to [the session management API](https://pkg.go.dev/go.viam.com/rdk/session#hdr-API).
+
+If you do this, you can manage the network operations of your robot manually, and operate your robot without the need for a consistent network connection.
+However, the low-network approach is not recommended if you want to utilize any of Viam's [services](/services/), like the vision service, motion planning, or the frame system.
+
+### How do Viam's client SDKs work if your robot loses connectivity during a session?
+
+As a client of your robot, the Python or TypeScript SDKs will have their connection to the Viam cloud and Viam app stopped when the robot loses its internet connection, if the session is ended properly.
+The Go Client SDK is a little bit different: as the session management API is implemented through the Go Client, the session can be managed from within this client, allowing for more flexibility in timing disconnection.
+
+Follow [these instructions](/program/apis/sessions/#use-the-session-management-api-to-manually-manage-sessions) to enable the manual session management.
+
+### How will your robot's primary parts and sub parts work if you lose connectivity during a session?
+
+Resources on the robot will still be connected and signaling to each other if your robot running `viam-server` is no longer connected to the internet and the session is ended, but they will not be connected to the Viam cloud and Viam app.
```

Comment:
```suggestion
The sub-parts of the robot will still be able to send operation requests to each other on a local network or directly (through the primary part) if your robot running `viam-server` is no longer connected to the internet and the session is ended, but they will not be connected to the Viam cloud and Viam app.
Because of the lack of connection to the cloud, they can no longer communicate with any of Viam's services, or interface with the app and appear as connected online.
```
@Otterverse That makes sense, but I don't know how to phrase this better or if I should expand this section. Would this suggestion help clarify/would this be accurate? My apologies if I'm still confused 🤣

Reply:
The section title may be the problem. If you want to talk about inter-part communication, then probably title the section as being about what happens if connections BETWEEN parts are lost. Otherwise, if those connections aren't lost, (only the incoming client connection for the robot itself) then the behavior is just the same as the previous section. E.g. sessions timeout, operations are halted.